packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -339,6 +342,25 @@ class TextPainter {
     markNeedsLayout();
   }
 
+
+  /// An optional minimum number of lines for the text to span.
```

Comment:
I don't think we should add `minLines` to `TextPainter`. We already have `TextPainter.preferredLineHeight` so computing the height for `minLines` shouldn't be that difficult without changing the `TextPainter` API. `TextPainter` is pretty much the lowest-level text public API, I'd say we should avoid adding convenience methods like this to keep the interface simple.

Also,  `minLines` doesn't make sense in a rich text context: the first line and the second line can have different line heights.